index.html

```diff
@@ -28,5 +33,129 @@ <h3 class="subtitle">I'm a Software Developer</h3>
     <li><a href="https://twitter.com/dafne_azzolina"><img src="icons/Twitter.png" alt="Twitter icon"></a></li>
     <li><a href="https://medium.com/"><img src="icons/Medium.png" alt="Medium icon"></a></li>
 </ul>
+</section>
+<!-- Beginning of second page -->
+<section class="secondPage">
+<section class="works">
+    <div>
+    <h2 class="workTitle">My recent works</h2>
+    <img src="css/Border.png" alt="Border" class="border"></div>
+   <div class="card1">
+       <img class="yoga" src="css/Works/Preview.png" alt="Preview"></div>
+   <div class="overflow">
+       <h2 class="bigText mainText">Multi-Post Stories</h2>
+       <p class="smallText mainText">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="socials">
+            <li class="boxes specialItem">CSS</li>
+            <li class="boxes specialItem">HTML</li>
+            <li class="boxes specialItem">Bootstrap</li>
+            <li class="boxes specialItem">Ruby</li>
+        </ul>
+   <button type="button" class="specialButton">See Project</button>
+    </div>
```

Comment:
- [ ] [OPTIONAL] I suggest sticking to the design by changing the background-color of this project to white